### PR TITLE
[Decoder] Implementation of image segmentation

### PR DESCRIPTION
**Pull request by niklasjang**
Added implementation of two functions.

1.  `set_label_index()` : Set a index with the highest probability of 21 labels for each pixel. 
2.  `set_color_according_to_label()` : Paint the appropriate color according to the label_idx for each pixel.

Signed-off-by: niklasjang <niklasjang@gmail.com>
Signed-off-by: Jihoon Lee <ulla4571@gmail.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Comment by myungjoo** [comment]
Please rebase

**Comment by myungjoo** [comment]
Do not add clutters. (no merge commits)

**Comment by jaeyun-jung** [comment]
plz fix line 18 also,
```
 * @file	tensordec-directvideo.c
```

**Comment by niklasjang** [comment]
I did rebase and changed file name in remark.

**Comment by jaeyun-jung** [comment]
TODO  (Next PR)
1. please consider to fix mem leak while doing caps negotiation in the function _init_modes().
2. set flag install true in meson script.

File: ext/nnstreamer/tensor_decoder/tensordec-imagesegment.c [hunk 1]
```diff
@@ -238,15 +238,59 @@ is_getTransformSize (void **pdata, const GstTensorsConfig * config,
...
+    0xFF808080, 0xFF4169E1, 0xFF008080, 0xFFFF6347, 0xFF000000,
+    0xFFFF4500, 0xFFDA70D6, 0xFFEEE8AA, 0xFF98FB98, 0xFFAFEEEE,
+    0xFFFFF5EE
+  };
+
+  for (i = 0; i < idata->height; i++) {
+    for (j = 0; j < idata->width; j++) {
+      int label_idx = idata->segment_map[i][j];
+      pos = &frame[i * idata->width + j];
+      if(label_idx == 0) continue; // Do not set color for background
```
Comment (myungjoo) [inline ext/nnstreamer/tensor_decoder/tensordec-imagesegment.c]: Please reformat with gst-indent script.

Comment (jaeyun-jung) [reply ext/nnstreamer/tensor_decoder/tensordec-imagesegment.c]: Also please fix line comment (c-std c89, use ```/* */```, do not use ```//```)

**Comment by niklasjang** [comment]
Three changes in force-push.
1. reformat with gst-indent
2. add g_assert()
3. use /* */ instead //

**Comment by myungjoo** [comment]
Great! I'll merge as soon as CI concludes.

**Pull request merged**